src/workerd/jsg/modules.h

```diff
@@ -434,6 +436,12 @@ class ModuleRegistryImpl final: public ModuleRegistry {
 
   void add(kj::Path& specifier, ModuleInfo&& info) {
     entries.insert(kj::heap<Entry>(specifier, Type::BUNDLE, kj::fwd<ModuleInfo>(info)));
+
+    if (entries.size() > 1) {
+      // This is just to give us some sense of how many workers make use of multiple modules
+      // in the bundle...
+      LOG_PERIODICALLY(WARNING, "NOSENTRY Worker using multiple modules in a single bundle.");
```

Comment:
will this happen once per isolate start or how frequently? would it be possible to aggregate this at a worker or an account level?

the main answer I'm trying to answer is: how many unique workers with multiple modules are active in a given period of time.